packages/data-point/lib/reducer-types/reducer-helpers/reducer-filter/resolve.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-env jest */
+'use strict'
+
+const Factory = require('./factory')
+const Resolve = require('./resolve')
+
+const Reducer = require('../../index')
+
+const DataPoint = require('../../../index')
+
+const AccumulatorFactory = require('../../../accumulator/factory')
+
+let manager
+
+beforeAll(() => {
+  manager = DataPoint.create()
+})
```

Comment:
The `let` and `beforeAll` might be better off inside the `describe('ReducerFilter#resolve'` block so that if we add another `describe` block in the future those will be scoped to these tests.